R/imports.R

```diff
@@ -0,0 +1,3 @@
+#' @importFrom glue glue
+#' @importFrom rlang abort
```

Comment:
This doesn't relate to this PR -- I should probabbly extract it to a separate PR.

Reply:
agreed

Reply:
https://github.com/2DegreesInvesting/PACTA_analysis/pull/178